snooty.toml

```diff
@@ -14,12 +14,14 @@ toc_landing_pages = [
   "/authentication",
   "/authentication/providers",
   "/mongodb",
+  "/getting-started",
   "/graphql",
   "/functions",
   "/functions/mongodb",
   "/services",
   "/services/http",
   "/services/twilio",
+  "/tutorials",
```

Comment:
```suggestion
```

remove this. we don't want it to show up as a landing page. just if a user happens to stumble upon the page, we want it to have a bit of content.